ExampleMod/Projectiles/ExampleLaser.cs

```diff
@@ -111,21 +119,23 @@ public override void AI()
 			{
 				Vector2 diff = mousePos - player.Center;
 				diff.Normalize();
-				projectile.position = player.Center + diff * _moveDist;
-				projectile.timeLeft = 2;
-				int dir = projectile.position.X > player.position.X ? 1 : -1;
-				player.ChangeDir(dir);
-				player.heldProj = projectile.whoAmI;
-				player.itemTime = 2;
-				player.itemAnimation = 2;
-				player.itemRotation = (float)Math.Atan2(diff.Y * dir, diff.X * dir);
-				#endregion
+                projectile.velocity = diff;
+                projectile.netUpdate = true;
 			}
-
-
-			#region Charging process
-			// Kill the projectile if the player stops channeling
-			if (!player.channel)
+            projectile.position = player.Center + projectile.velocity * MOVE_DISTANCE;

```

Comment:
Did you take this code out of the **if (projectile.owner == Main.myPlayer)** statement? That's exactly what should make it work properly in multiplayer, you only want the projectile pos and player rotation to update if the current player is the owner.